apps/judicial-system/web/src/components/FeatureProvider/FeatureProvider.tsx

```diff
@@ -6,7 +6,10 @@ interface FeatureProvider {
   features: Feature[]
 }
 
-const availableFeatures: Feature[] = [Feature.INDICTMENTS]
+const availableFeatures: Feature[] = [
```

Comment:
We may be able to simplify this by using something like `Object.keys(Feature)`. That will make this less error prone.

Reply:
Excellent point! I do think, however that belongs in a separate PR since this is not straight forward. The type of Object.keys will always be string[], and therefore we'd miss the typing on this. I'd like to dig a little into typescript to do this properly. I've added a task to our Inbox.